arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+        },
+        "EnableDeadLetteringOnFilterEvaluationExceptions": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support on Filter evaluation exceptions."
+        },
+        "EnableDeadLetteringOnMessageExpiration": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support when a message expires."
+        },
+        "ExtensionData": {
+          "type": "object",
+          "description": "Structure that contains extra data."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+        },
+        "IsReadOnly": {
+          "type": "boolean",
+          "description": "Value that indicates whether the entity description is read-only."
+        },
+        "LockDuration": {
+          "type": "string",
+          "format": "time-span",
+          "description": "The lock duration time span for the subscription."
+        },
+        "MaxDeliveryCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of maximum deliveries."
+        },
+        "MessageCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Number of messages."
+        },
+        "MessageCountDetails": {
+          "type": "object",
+          "description": "Message details for a subscription."
+        },
+        "Name": {
+          "type": "string",
+          "description": "Name of the subscription description."
+        },
+        "RequiresSession": {
+          "type": "boolean",
+          "description": "Value indicating if a subscription supports the concept of session."
+        },
+        "EntityStatus": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of a messaging entity.",
+          "enum": [
+            "Active",
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "ReceiveDisabled",
+            "Renaming",
+            "Restoring",
+            "SendDisabled",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": false
+          }
+        },
+        "TopicPath": {
+          "type": "string",
+          "description": "Path of the topic that this subscription description belongs to."
+        },
+        "UpdatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The exact time the message has been updated."
+        },
+        "UserMetadata": {
+          "type": "string",
+          "description": "The user metadata."
```

Comment:
The description is not helpful. What would the value be like. It would be great if an example is provided, for the customer to comprehend.